src/main/resources/reference.conf

```diff
@@ -8,8 +8,17 @@ kafka-lag-exporter {
   client-group-id = "kafkalagexporter"
   client-group-id = ${?KAFKA_LAG_EXPORTER_CLIENT_GROUP_ID}
   kafka-client-timeout = 10 seconds
-  kafka-client-timeout =  ${?KAFKA_LAG_EXPORTER_KAFKA_CLIENT_TIMEOUT_SECONDS}
-  clusters = []
+  kafka-client-timeout = ${?KAFKA_LAG_EXPORTER_KAFKA_CLIENT_TIMEOUT_SECONDS}
+  clusters = [
```

Comment:
The `reference.conf` serves as default configuration for the project.  This will overridden when using the Helm chart, but for people running in standalone mode it may be applied if they don't override it with their own `application.conf`.  This should be reverted to `clusters = []`.